hack/with-unpinned-kust-dev.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# Copyright 2023 The Kubernetes Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -x
+set -e
+set -o pipefail
+set -o nounset
+
+# This script uses 'replace' statements to 'unpin' local modules from module versions go.mod normally
+# requires, so that the local version will be used instead. With the advent of Workspace mode, we no longer
+# need to do this in general in between releases. However, some key commands like `go mod tidy` are not
+# Workspace-aware and thus will fail if API changes between modules exist on master. This script allows us to
+# test those commands without requiring unpin operations in our release workflow.
+
+if [[ -z "${1-}" ]] ; then
+  echo "Usage: $0 <cmd>"
+  echo "Example: $0 'go mod tidy -v'"
+  exit 1
+fi
+
+cmd=$1
+
+# First we read in the list of all kustomize modules and their local locations. The data looks like:
+# sigs.k8s.io/kustomize/api /Users/you/src/sigs.k8s.io/kustomize/api
+# sigs.k8s.io/kustomize/cmd/config /Users/you/src/sigs.k8s.io/kustomize/cmd/config
+IFS=$'\n'
+modules=($(go list -m -f "{{.Path}} {{.Dir}}"))
```

Comment:
Brilliant! Thank you, @KnVerey! Your code led me to the [package template docs](https://pkg.go.dev/text/template). I was missing the double quotes!